tests/integration/Mvc/Model/BinderCest.php

```diff
@@ -0,0 +1,500 @@
+
+        $dispatcher->dispatch();
+
+        $returnedValue = $dispatcher->getReturnedValue();
+        $I->assertInstanceOf('Phalcon\Test\Models\People', $returnedValue);
+        $I->assertEquals($people->cedula, $returnedValue->cedula);
+        $I->assertEquals([
+            'people' => 'Phalcon\\Test\\Models\\People',
+        ], $this->cache->get('_PHMB_Test10Controller_viewAction'));
+
+        $robot = Robots::findFirst();
+
+        $dispatcher->setActionName('multiple');
+        $dispatcher->setParams(['people' => $people->cedula, 'robots' => $robot->id]);
+
+        $dispatcher->dispatch();
+
+        $returnedValue = $dispatcher->getReturnedValue();
+        $I->assertInstanceOf('Phalcon\Test\Models\People', $returnedValue[0]);
+        $I->assertInstanceOf('Phalcon\Test\Models\Robots', $returnedValue[1]);
+        $I->assertEquals($people->cedula, $returnedValue[0]->cedula);
+        $I->assertEquals($robot->id, $returnedValue[1]->id);
+        $I->assertEquals([
+            'people' => 'Phalcon\\Test\\Models\\People',
+            'robots' => 'Phalcon\\Test\\Models\\Robots',
+        ], $this->cache->get('_PHMB_Test10Controller_multipleAction'));
+
+        $dispatcher->setControllerName('test11');
+        $dispatcher->setActionName('view');
+        $dispatcher->setParams(['people' => $people->cedula]);
+
+        $dispatcher->dispatch();
+
+        $returnedValue = $dispatcher->getReturnedValue();
+        $I->assertInstanceOf('Phalcon\Test\Models\People', $returnedValue);
+        $I->assertEquals($people->cedula, $returnedValue->cedula);
+        $I->assertEquals([
+            'people' => 'Phalcon\\Test\\Models\\People',
+        ], $this->cache->get('_PHMB_Test11Controller_viewAction'));
+
+        $dispatcher->setActionName('multiple');
+        $dispatcher->setParams(['people' => $people->cedula, 'robots' => $robot->id]);
+
+        $dispatcher->dispatch();
+
+        $returnedValue = $dispatcher->getReturnedValue();
+        $I->assertInstanceOf('Phalcon\Test\Models\People', $returnedValue[0]);
+        $I->assertInstanceOf('Phalcon\Test\Models\Robots', $returnedValue[1]);
+        $I->assertEquals($people->cedula, $returnedValue[0]->cedula);
+        $I->assertEquals($robot->id, $returnedValue[1]->id);
+        $I->assertEquals([
+            'people' => 'Phalcon\\Test\\Models\\People',
+            'robots' => 'Phalcon\\Test\\Models\\Robots',
+        ], $this->cache->get('_PHMB_Test11Controller_multipleAction'));
+
+        // Reset dispatcher
+        $dispatcher = new Dispatcher;
+        $dispatcher->setDI(Di::getDefault());
+
+        $I->assertInstanceOf('Phalcon\Di', $dispatcher->getDI());
+        $I->haveServiceInDi('dispatcher', $dispatcher);
+
+        $dispatcher->setControllerName('test9');
+        $dispatcher->setActionName('view');
+        $dispatcher->setParams(['people' => $people->cedula]);
+
+        try {
+            $dispatcher->dispatch();
```

Comment:
to separate test... Probably it should named like "testDispatcherInstanceException" also you can use something 

```
$I->expectException()
```
[method](http://codeception.com/docs/modules/Asserts#expectException) in this test